### PR TITLE
alert\history query entityExpression filter

### PR DESCRIPTION
**Pull request by Sild**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/alert/AlertMethod.java [hunk 1]
```diff
@@ -66,4 +66,17 @@ public static void generateAlertForEntity(final String entityName) throws Except
...
 
+    public static void generateAlertHistoryForEntity(final String entityName) throws Exception {
+        Series series = new Series();
+        series.setEntity(entityName);
+        series.setMetric(Util.RULE_METRIC_NAME);
+        series.addData(new Sample(Util.ISOFormat(new Date()), Util.ALERT_OPEN_VALUE));
+        SeriesMethod.insertSeriesCheck(series);
+
+        series.setData(null);
+        series.addData(new Sample(Util.ISOFormat(new Date()), Util.ALERT_CLOSE_VALUE));
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/alert/AlertMethod.java]: Why do you use new Date() instead constant?

**Comment by rmakulov** [comment]
lgtm

File: src/test/java/com/axibase/tsd/api/method/alert/AlertMethod.java (hunk 1 above)
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/alert/AlertMethod.java]: ATSD rules will not apply to series which does not satisfy the following condition:
|currentDate - seriesDate| < VALUE
There VALUE is specified in ATSD (and is about 1 minute)

**Comment by unrealwork** [comment]
ok

**Pull request merged**